crates/ruma-common/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # [unreleased]
 
+Bug fixes:
+
+* Re-export `MatrixIdError`, `MatrixToError`, `MatrixUriError` and `MxcUriError`
```

Comment:
I think this sounds a bit like you previously just had to import from the original location. Maybe sth. like "expose" (types...) "at the crate root" is clearer?